lua/entities/gmod_tardis/modules/sh_falling.lua

```diff
@@ -0,0 +1,191 @@
+if SERVER then
+    function ENT:IsVerticalLanding(collision_data)
+        local ang = self:GetAngles()
+        local angmax = math.max(math.abs(ang.p), math.abs(ang.r))
+
+        return (angmax < 30 and collision_data.HitNormal.z < -0.85 and collision_data.OurOldVelocity.z < 0)
```

Comment:
I think we need to take into account horizontal velocity here also as you can do a ridiculously fast landing and just stop instantly

Reply:
Also if there is a reasonable amount of angular velocity because if you end up in a spin or doing flips or something and just so happen to hit the ground well enough it will also instantly stop which feels wrong

Reply:
Also also I found that you can land on a very slight hill and keep sliding down slightly every time the brakes disengage and it gets in a bit of a loop until it finally settles, so I think we should also ignore very slow downward velocity

Reply:
we have a "TARDIS flip challenge" now, lol

What alternative behaviour do you propose for landing like that?

I definitely don't want the TARDIS bouncing around again, that never looks realistic, it aint a rubber ball. Judging by the first TARDIS scene of "deep breath", the TARDIS can instantly stabilize itself vertically after a very horizontal fall, as well as consuming the impact from a really high one ("NOTD"). 

Currently, the fall is not always vertical (as it used to be in the first version), but when it is vertical, the landing happens, and the drum makes it clear that some sort of physical brake is in place. So even though it might not be realistic with props, imo it fits the behaviour a TARDIS should have

Reply:
I'll see what I can do about sliding down

Reply:
I think as long as the horizontal velocity isn't stupidly high and the angular velocity is also not high then we can continue to let it do the landing. It should be able to stop from a small spin or small horizontal velocity I agree but too much is a jarring experience

Reply:
#include <discord_discussion.h>

e91a74397c27d5b0cc8d5ba5d93c2cb3637f6940

Sliding should be fixed in 0f10c6662c3cc5218eb2182547cb05b1d2cdcad9